fml/platform/fuchsia/message_loop_fuchsia.cc

```diff
@@ -30,8 +32,11 @@ void MessageLoopFuchsia::WakeUp(fml::TimePoint time_point) {
     due_time = zx::nsec((time_point - now).ToNanoseconds());
   }
 
-  auto status = async::PostDelayedTask(
-      loop_.dispatcher(), [this]() { RunExpiredTasksNow(); }, due_time);
+  if (task_.is_pending()) {
+    task_.Cancel();
```

Comment:
Is accessing the task from multiple threads safe to do? I am not sure. If it is, then this works fine.